Command/RunCommand.php

```diff
@@ -428,9 +429,25 @@ private function getCommandProcessBuilder()
             $pb->add('exec');
         }
 
+        // Localize the `console` command
```

Comment:
@Ener-Getick @schmittjoh what do you think of https://github.com/Ener-Getick/JMSJobQueueBundle/pull/3 ?